init.lua

```diff
@@ -222,3 +222,21 @@ minetest.register_chatcommand("whereis", {
 				pos.x, pos.y, pos.z)
 	end
 })
+
+minetest.register_chatcommand("s", {
+    params = "<msg>",
+    description = "Send a message on the staff channel",
+    privs = {kick=true},
```

Comment:
Spaces around `=` and within braces improve readability.

```suggestion
    privs = { kick = true },
```